### PR TITLE
Update Quasar to mainnet

### PR DESCRIPTION
**Pull request by dylanschultzie**
@pabloqsr please review

### CODE REVIEW

File: quasar/chain.json [hunk 1]
```diff
@@ -19,128 +20,71 @@
       }
     ]
   },
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ibc/0471F1C4E7AFD3F07702BEF6DC365268D64570F7C1FDC98EA6098DD6DE59817B",
```
Comment (JeremyParish69) [inline quasar/chain.json:26]: I don't know what these three tokens are, but could we register them as ics20 assets and add register their corresponding ibc channels?

File: quasar/chain.json [hunk 2]
```diff
@@ -19,128 +20,71 @@
...
+        "fixed_min_gas_price": 0.01
+      },
+      {
+        "denom": "ibc/FA0006F056DB6719B8C16C551FC392B62F5729978FC0B125AC9A432DBB2AA1A5",
+        "fixed_min_gas_price": 0.01
+      },
+      {
+        "denom": "ibc/FA7775734CC73176B7425910DE001A1D2AD9B6D9E93129A5D0750EAD13E4E63A",
```
Comment (dylanschultzie) [inline quasar/chain.json:34]: axlUSDC (from osmosis?)

File: quasar/chain.json (hunk 1 above)
Comment (dylanschultzie) [inline quasar/chain.json:26]: OSMO

File: quasar/chain.json [hunk 3]
```diff
@@ -19,128 +20,71 @@
...
     ]
   },
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ibc/0471F1C4E7AFD3F07702BEF6DC365268D64570F7C1FDC98EA6098DD6DE59817B",
+        "fixed_min_gas_price": 0.01
+      },
+      {
+        "denom": "ibc/FA0006F056DB6719B8C16C551FC392B62F5729978FC0B125AC9A432DBB2AA1A5",
```
Comment (dylanschultzie) [inline quasar/chain.json:30]: ATOM

File: quasar/chain.json (hunk 1 above)
Comment (pabloqsr) [reply quasar/chain.json:26]: Hey @JeremyParish69 
Chain is only accepting those denoms for gas. Native token will be launch after a few weeks.
How do we register them as ics20 assets?

We will be also registering the corresponding IBC channels soon when we visit

Comment (dylanschultzie) [reply quasar/chain.json:26]: @pabloqsr we ended up discussing adding the ICS tokens off-line; you can see them in the assets file. No more is needed here at this time

**Pull request merged**